src/collidermodus.cc

```diff
@@ -122,6 +128,45 @@ static constexpr int LCollider = LogArea::Collider::id;
  * additional specification of \key Beta_2, \key Beta_4, \key Theta and
  * \key Phi, which follow \iref{Moller:1993ed} and \iref{Schenke:2019ruo}. \n
  *
+ * \li \key E_Kin (double, optional, no default) Set the kinetic
+ * energy (in GeV) per particle of the beam.  This key, if used, must
+ * be present for both \key Projectile and \key Target.
+ *
+ * \li \key P_Lab (double, optional, no default) Set the momentum (in
+ * GeV/c) per particle of the beam.  This key, if used, must be
+ * present for both \key Projectile and \key Target.
+ 
+ * \note Note on \key E_Kin and \key P_Lab.  If the beam specific
+ * kinetic energy or momentum is set using either of these keys, then
+ * it must be specified in the same way (not necessarily same value)
+ * for both beams.  This is useful to simulate for example p-Pb
+ * collisions at the LHC where the centre-of-mass system does not
+ * correspond to the laboratory system.  E.g.,
+ *
+ *\verbatim
+Modi:
+  Collider:
+    Calculation_Frame: center of velocity
+    Impact:
+      Random_Reaction_Plane: True
+      Range: [0, 8.5]
+    Projectile:
+      E_Kin: 1580
+      Particles:
+        2212: 82
+        2112: 126
+    Target:
+      E_Kin: 4000
+      Particles:
+        2212: 1
+        2112: 0
+ \endverbatim
+ *
+ * Note that SMASH performs it's calculation in the centre-of-velocity
+ * and the particles are returned in the centre-of-mass frame.  The
+ * particles therefore need to be boosted by the rapidity of the
+ * centre-of-mass (-0.465 for p-Pb at 5.02TeV).
+ *
```

Comment:
Above are some mix-ups when our documentation is build from the comments. I will take care of this myself.